sklearn/ensemble/tests/test_partial_dependence.py

```diff
@@ -128,27 +127,30 @@ def test_partial_dependecy_input():
     clf = GradientBoostingClassifier(n_estimators=10, random_state=1)
     clf.fit(X, y)
 
-    assert_raises(ValueError, partial_dependence,
-                  clf, [0], grid=None, X=None)
+    with pytest.raises(ValueError):
```

Comment:
This whole file will be deleted soon since the PDPs are deprecated here, so no need to change it